guides/common/modules/proc_creating-a-webhook-template.adoc

```diff
@@ -0,0 +1,12 @@
+[id="creating-a-webhook-template_{context}"]
+= Creating a Webhook Template
+
+.Procedure
+
+. In the {Project} web UI, navigate to *Administer* > *Webhooks Templates*.
```

Comment:
```suggestion
. In the ProjectWebUI, navigate to *Administer* > *Webhooks Templates*.
```
We have a proper attribute for this: :ProjectWebUI: {https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L84
Please apply throughout.